toc_subprojects/project-reviews-subproject/governance-review-template.md

```diff
@@ -1,222 +1,208 @@
+<br>
+**Incubating:** Required | **Graduated:** Required
 
-* Document a complete maintainer lifecycle process (including roles, onboarding, offboarding, and emeritus status).
-* Demonstrate usage of the maintainer lifecycle with outcomes, either through the addition or replacement of maintainers as project events have required.
 
-<!--- Look at governance.md/contributor ladder and other documents.  Look at maintainer.md file history.
-Look for evidence that maintainers have turned over, particularly new maintainer recruitment
-and retiring inactive maintainers.  Also look that qualifications for new maintainers are
-clear and that those qualifications are honored. --->
+**A number of active maintainers which is appropriate to the size and scope of
+the project.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
+
+**Project maintainers from at least 2 organizations that demonstrates
+survivability.**
+<br>
+**Incubating:** N/A | **Graduated:** Required
 
-* Project maintainers from at least 2 organizations that demonstrates survivability.
 
-<!--- Look at maintainer employer affiliation. At graduated level, having everyone working for the same
-organization demonstrates some probable problems.  Inactive maintainers do not fix the maintainer
-diversity problem. Combine this review with lifecycle above; projects with strong recruitment stories
-are lower risk even when not very diverse. --->
 
 ### Ownership
 
-* Code and Doc ownership in Github and elsewhere matches documented governance roles.
+**Code and Doc ownership in Github and elsewhere matches documented governance
+roles.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
 
-<!--- right now there is very little ability to audit this.  You can record here 
-whether the project is using Prow or Sherriff (both of which are good), but if not
-you can't verify ownership. Parts of the governance documentation that talk about
-assigning permissions is also good, but the absence of those isn't notable. --->
 
 ### Code of Conduct
 
-* Document adoption of the CNCF Code of Conduct
-* CNCF Code of Conduct is cross-linked from other governance documents.
+**Document adoption of the CNCF Code of Conduct**
+<br>
+**Incubating:** Required | **Graduated:** Required
 
-<!--- The CNCF CoC should be present in all appropriate places in the project, which 
-includes their Community or main repository, and public website at a minimum.  Other
-community and governance documentation should refer to the CoC; for example, role
-removal process should mention that people can be removed for CoC violations.  
-If the project is handling their own CoC reports rather than just sending them 
-directly to the CNCF CoCC, you need to check if this is adequately documented, including
-who receives the reports and how and where they're evaluated. --->
 
-### Subprojects
+**CNCF Code of Conduct is cross-linked from other governance documents.**
```

Comment:
```suggestion
**The Project's Code of Conduct (which is the CNCF CoC or based off it) is cross-linked from other governance documents.**
```

Reply:
I'm leaving as is for now, if we want to update the wording, then the wording should be updated across ALL docs that had the question, and not just the one in this template